app/scripts/migrations/index.js

```diff
@@ -180,7 +180,7 @@ const migrations = [
   require('./151'),
   require('./152'),
   require('./153'),
-  require('./154'),
+  require('./152.1'),
```

Comment:
should 152.1 come before 153?

Reply:
nit ☝️

Reply:
https://github.com/MetaMask/metamask-extension/pull/32109/commits/a51e2b88bab51a6b48fe81c4f45d97332292e794